src/fullyautomatednutcracker/cogs/antiselfdeprecation.py

```diff
@@ -29,5 +32,16 @@ async def on_message(self, message):
         if message.content.lower() == 'yoshi man good':
             await message.add_reaction('\U0001F49A')
 
+    # bump timer, waits 2 hours and 30 minutes
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author.id == 302050872383242240 and 'Bump done' in message.embeds[0].description:
+            bumped = time.monotonic()
+            self.last_bumped = bumped
+            await message.add_reaction(':thumbsup:')
+            await asyncio.sleep(9000)
+            if self.last_bumped == bumped:
+                await message.channel.send('bump timer has expired')
```

Comment:
we should make this more inviting... perhaps:

":spongeomg: No one's bumped our server in over two hours! Disboard keeps us up on the front page! Use `!d bump` to bump us!"